.github/workflows/snyk.yml

```diff
@@ -0,0 +1,29 @@
+# This action runs every day at 6 AM and on every push
+# If the branch it's running on is main then it will run snyk monitor (reports vulnerabilities to snyk.io)
+# Otherwise it will run snyk test
```

Comment:
is this comment still accurate? as far as I can tell the workflow is only run on pushes to main, and always runs snyk monitor

Reply:
Good spot! Will update...